### ISSUE TITLE
FindAndCountAll() counts included models too?

### ISSUE THREAD

**Issue by dschwertfeger** [sequelize/sequelize#1773]
I noticed that `findAndCountAll` seems to count included models, too. I had a look into the code and it looks like `include` is not omitted from `findOptions`. 

Is this the desired behavior?

It looks like it had been different in the past. In #1202 `include` was removed from the attributes to be omitted. Why was it removed? I can't really think of a case in which you'd want to count associated models, too, but maybe that's because I really don't need this in what I'm trying to do. Is there a case in which it makes sense to count the `include`d models as well? For me it just totally throws off pagination.

How would I get only the count of the Records in the following example? Is there a workaround to somehow manually omit the `include` attribute?

``` coffee
limit = 50
offset = limit * (page - 1)
Record.findAndCountAll
  limit: limit
  include: [Label, Image]
  offset: offset
.success (result) ->
  res.render 'records/index',
    success: req.flash 'success'
    records: result.rows
    page: page
    pages: Math.ceil(result.count / limit)
```

**Comment by billxinli** [comment]
I am having similar issue with this problem where FindAndCountAll counts with a left outer join, and as such, it counts all other included models.

Will there be a fix for 1.7?

**Comment by janmeier** [comment]
@billxinli 2.0 has been officially released, so the core team will work exclusively on that for now. 

We will still accepts PRs, but that's it
